EternalBAND.DomainObjects/ApiContract/LoginOutputContract.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace EternalBAND.DomainObjects.ApiContract
+{
+    public class LoginOutputContract
+    {
+        public string Token { get; set; }
+        public string ReturnUrl { get; set; }
```

Comment:
validation check 2 side, fe and be. Expire time decision

Reply:
We may integrate a flow right before the Request and basically add minimal validation rule. LoginOutputContract is output contract so no need for validation but for input contract like  "LoginInputContract" we can add this.

Reply:
- Yes and lets add the expire time in here. 
---
- Also yes validations for inputs. It will add for these.
![image](https://github.com/user-attachments/assets/145b8270-befe-4e27-98c0-df15e6eb8e1a)

Reply:
No need to hold expiration time of token as a separated property here, each endpoint may read this info from Authorization header on fly, also no need to.add manual expiry control , .net itself is managing it already.

I am planning to use generic types here so each validator can be usable according to contract data type 

IValidator<TType> and we can register as 

UserValidator as IValidator<User>

On the runtime we may instantiate related validator and use generic function call

Reply:
Yes it is true BE side dotnet is checking expire time with automatically, we will not validate from here for BE. But FE side we need this property, so we can keep it on our user claims or we can set directly login response. So the end of the day we need this one for the FE.

And yes as I mentioned in whatsapp, generic validator is more useful then others. Also FE will be use generic validators. Also annotations for help the validating process. In Spring Boot it is include,  maybe dotnet has also these kind of helping annotations.

Reply:
We decided. it is keeping automatic in claims(with a new JWT versions). So there is no need to set any expire time to somewhere.

Reply:
contract base  validation is added, fluent validator is being used as a validator library